runtime/precompiles/src/staking.rs

```diff
@@ -81,10 +80,16 @@ where
 			[0x85, 0x45, 0xc8, 0x33] => {
 				return Self::is_candidate(&input[SELECTOR_SIZE_BYTES..]);
 			}
-			// c9f593b2
+			[0x8f, 0x6d, 0x27, 0xc7] => {
+				return Self::is_selected_candidate(&input[SELECTOR_SIZE_BYTES..]);
+			}
 			[0xc9, 0xf5, 0x93, 0xb2] => {
+				//TODO Do we need to verify that there were no additional bytes passed in here?
```

Comment:
I don't think so. There is a unique 4 byte accessor for `min_nomination`. Any call prefixed with these bytes should be treated as a call to `min_nomination`. There will never be a case when a call is prefixed with these bytes and more data is used to express context.